kani-driver/src/assess.rs

```diff
@@ -0,0 +1,226 @@
+// Copyright Kani Contributors
+// SPDX-License-Identifier: Apache-2.0 OR MIT
+
+use std::{collections::HashMap, path::PathBuf};
+
+use anyhow::Result;
+use comfy_table::Table;
+use kani_metadata::KaniMetadata;
+
+use crate::harness_runner::HarnessResult;
+use crate::session::KaniSession;
+
+/// Set some defaults for how we format tables
+fn assess_table_new() -> Table {
+    use comfy_table::*;
+
+    let mut table = Table::new();
+    table.set_content_arrangement(ContentArrangement::Dynamic);
+    table
+        .load_preset(comfy_table::presets::NOTHING)
+        .set_style(TableComponent::BottomBorder, '=')
+        .set_style(TableComponent::BottomBorderIntersections, '=')
+        .set_style(TableComponent::TopBorder, '=')
+        .set_style(TableComponent::TopBorderIntersections, '=')
+        .set_style(TableComponent::HeaderLines, '-')
+        .set_style(TableComponent::MiddleHeaderIntersections, '+')
+        .set_style(TableComponent::VerticalLines, '|');
+    table
+}
+
+/// Internal data type for constructing the unsupported features table
+#[derive(Default)]
+struct UnsupportedFeaturesTableData {
+    crates_impacted: usize,
+    instances_of_use: usize,
+}
+fn unsupported_features_table(metadata: &KaniMetadata) -> Table {
```

Comment:
I'd prefer we call this `generate_unsupported_features_table` or `build_unsupported_features_table`. Same goes for all other functions in this module. They'd also benefit from some function-level documentation, adding an example of what the table looks like and what the values in there mean.